po/zh_CN.po

```diff
@@ -1076,24 +1095,27 @@ msgid ""
 "length must either be 1 (single values are ok) or match the LHS length "
 "exactly. If you wish to 'recycle' the RHS please use rep() explicitly to "
 "make this intent clear to readers of your code."
-msgstr ""
+msgstr "提供 %d 项给第 %d 列的第 %d 组，这组有 %d 行"
+"RHS長度必須為 1 (可以使用單個值) 或與 LHS 長度完全匹配"
+"如果您想回收(recycle) RHS，请使用 rep() 向你的代码读者明确表达你的意图"
 
 #: dogroups.c:444
 #, c-format
 msgid "Wrote less rows (%d) than allocated (%d).\n"
-msgstr ""
+msgstr "写入的行 (%d) 少于分配的 (%d)\n"
```

Comment:
nbd but missing a period

Reply:
Thanks for your comment, but I don`t really understand what you mean.

Reply:
nbd means "no big deal". after `(%d)` you're missing `.`

Reply:
got that. I forgot period also mean `.`.